notification-templates/job_failed.html

```diff
@@ -211,7 +211,9 @@
                             </tr>
                             <tr>
                                 <td align="center" style="font-family: 'Montserrat', Arial, sans-serif; color:#333F52; font-size:20px; line-height: 24px; letter-spacing:1px;">
-                                    For help troubleshooting, see our <a href="https://broadscientificservices.zendesk.com/hc/en-us/articles/39901328838427">documentation</a>. If you still need help, email <a href="mailto:scientific-services-support@broadinstitute.org">scientific-services-support@broadinstitute.org</a>.
+                                    View your jobs in the <a href="https://services.terra.bio/#pipelines/imputation/history">Scientific Services UI</a>.<br><br>
+                                    For help troubleshooting, see our <a href="https://broadscientificservices.zendesk.com/hc/en-us/articles/39901328838427">documentation</a>.<br><br>
```

Comment:
I think the breaklines after the "documentation" link make the email look a little weird, i like how the original email was where the text was wider than the table with info